source/gui/exit.py

```diff
@@ -0,0 +1,131 @@
+def quit():

```

Comment:
One [suggestion ](https://github.com/nvaccess/nvda/pull/13498#discussion_r832661603) was to change the name. I tend to agree that `gui.exit.quit` is confusing. Lets change this to a better name (`triggerExitCommand`?) and add some docs.

Reply:
`gui.exit.quit` was removed in cf35dc1  as per the discussion in https://github.com/nvaccess/nvda/pull/13498#discussion_r832739656